aws/resource_aws_glue_catalog_table.go

```diff
@@ -271,6 +276,15 @@ func resourceAwsGlueCatalogTableRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error reading Glue Catalog Table: %s", err)
 	}
 
+	tableArn := arn.ARN{
+		Partition: meta.(*AWSClient).partition,
+		Service:   "glue",
+		Region:    meta.(*AWSClient).region,
+		AccountID: meta.(*AWSClient).accountid,
```

Comment:
Can you please add this resource to the `skip_requesting_account_id` documentation in `website/docs/index.html.markdown`? Thanks!

Reply:
done